frontend/src/components/object/ObjectTable.vue

```diff
@@ -127,9 +127,19 @@ const filters = ref({
               v-model="filters['global'].value"
```

Comment:
minor point but to avoid over-powering css, it might be worth adding a class to this element so the css rule below is more targeted:
`<InputText class="searchInput"..`
...
then on line 297:
```css
:deep(.searchInput .p-inputtext) { /* might be .searchInput:deep(p.inputtext) { */
  padding-right: 2.5rem;
}
```